packages/material-ui/src/Chip/Chip.js

```diff
@@ -209,6 +210,11 @@ Chip.propTypes = {
    * @ignore
    */
   className: PropTypes.string,
+  /**
+   * If `true`, the chip will be clickable.
+   * By default, the chip will not be clickable.
+   */
```

Comment:
Assuming we keep the current logic, the wording needs to be something like:

"If `true`, the chip will appear clickable and hover when pressed, even if the `onClick` property is not defined. This can be used, for example,  along with the `component` property to indicate an anchor Chip is clickable."

Reply:
And an example in the docs would be good...